include/matrix3x4SIMD.h

```diff
@@ -213,12 +213,20 @@ struct matrix3x4SIMD
 
 	inline void transformVect(float* _out, const float* _in) const
 	{
-		vectorSIMDf c0 = rows[0], c1 = rows[1], c2 = rows[2], c3(0.f, 0.f, 0.f, 1.f);
-		core::transpose4(c0, c1, c2, c3);
+		vectorSIMDf vec(_in);
+		vectorSIMDf r0 = rows[0] * vec, 
+			r1 = rows[1] * vec, 
+			r2 = rows[2] * vec, 
+			r3 = vec & BUILD_MASKF(0, 0, 0, 1);
```

Comment:
with 3 hadds this will go in the res[3] element which will never be memcopied out... as this is the non 4x1 function maybe for this specific instance (mul with 3x1 vector) you should use the default zero vector for r3